src/axom/core/utilities/About.cpp.in

```diff
@@ -6,21 +6,23 @@
 #include "axom/config.hpp"
 #include "axom/core/utilities/About.hpp"
 
-#include <algorithm>  // copy
-#include <iostream>   // for cout
-#include <sstream>    // for stringstream
-#include <string>     // for C++ string
-#include <vector>     // for STL vector
-#include <iterator>   // for ostream_iterator
+#include <algorithm>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <iterator>
 
 namespace axom
 {
-//-----------------------------------------------------------------------------
+
+std::string gitSHA()
+{
+  return "@AXOM_GIT_SHA@";
```

Comment:
Suggestion: It might be good to add a note/comment to this file indicating why this is a config file and when `AXOM_GIT_SHA` gets updated.

Reply:
Added.